src/datahike/api.cljc

```diff
@@ -203,9 +203,7 @@
                                                   {:error :transact/syntax
                                                    :argument-keys (keys arg-map)}))
                 (or (vector? arg-map)
-                    (seq? arg-map))   (do (log/warn "Vector and sequence as argument are deprecated."
-                                                    "Instead use a map with key :tx-data.")
-                                          {:tx-data arg-map})
+                    (seq? arg-map))   {:tx-data arg-map}
```

Comment:
in api-ns there are docstrings where the deprecation is mentioned. that should be removed then as well.

Reply:
and the deprecation is also mentioned in connector.cljc line 145

Reply:
I have improved the doc string for `api/transact`, but I don't really see where deprecation is indicated there. `connector.cljc:145` points to unrelated code for me. Which code did you mean?

Reply:
I think, he is talking about the location where you copied this from.

Reply:
sorry, my bad.